### PR TITLE
fix: CVE issues

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/security/issues/16

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/longhorn/longhorn-instance-manager
 
-go 1.20
+go 1.21
+
+replace github.com/longhorn/go-spdk-helper => github.com/c3y1huang/go-spdk-helper v0.0.0-20231017050105-6c08295fcd89
```
Comment (c3y1huang) [inline go.mod:5]: TODO: update after https://github.com/longhorn/go-spdk-helper/pull/48 is merged.

**Comment by c3y1huang** [comment]
[Result before and after the fix.](https://github.com/longhorn/security/issues/16#issuecomment-1765679338)

**Comment by c3y1huang** [comment]
ref https://github.com/longhorn/longhorn/issues/7001

**Review by innobead — APPROVED** [review]

**Comment by c3y1huang** [comment]
@mergify backport v1.5.x v1.4.x

**Pull request merged**